static/app/views/dashboards/widgetBuilder/hooks/useWidgetBuilderState.tsx

```diff
@@ -562,6 +562,15 @@ function useWidgetBuilderState(): {
         }
         case BuilderStateAction.SET_Y_AXIS:
           setYAxis(action.payload, options);
+
+          if (fields?.length && fields.length > 0) {
```

Comment:
This is here because of the bug noted in `widgetCanUseTimeSeriesVisualization.tsx`, updating the yAxes wouldn't re-calculate the limit, so a request would fire, but then something else downstream fixed the limit and would fire another request causing some weird cancellation state that broke the new timeseries.